helper

```diff
@@ -0,0 +1,7 @@
+apt-get update; apt-get install -y git vim sudo python3 build-essential; \
```

Comment:
this `helper` script is not intended to be kept here right?

Reply:
yes